src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolStcicJudicialRoleMappingTest.java

```diff
@@ -49,12 +49,16 @@ class DroolStcicJudicialRoleMappingTest extends DroolBase {
         expectedRoleNameWorkTypesMap.put("fee-paid-financial", null);
     }
 
-    static void assertCommonRoleAssignmentAttributes(RoleAssignment r) {
+    static void assertCommonRoleAssignmentAttributes(RoleAssignment r, String office, List<String> ticketCodes) {
```

Comment:
should we can omit `office` & `ticketCodes` from input param as not used in test?